### ISSUE TITLE
[Improvement] Checking the movement of the model vector and hitboxes in the taunt state.

### ISSUE THREAD

**Issue by WOLFA22** [sapphonie/StAC-tf2#155]
The gist is as follows:

If a player, being in a taunt state - turned sharply to the other side or several times in a second (SARACULARLY, not smoothly!) - detection. Or the movement vector of the player in the taunt does not coincide with the animation.

This feature is used in private LMAOBOX. The essence is that the model goes straight, according to the animation, but the hitboxes go somewhere sideways. 

And as a result it turns out - the player in mock state goes animated in one direction, but moves in the opposite direction.

If there will be a video showing it - I'll throw up, it's easier to see once than read a text from an interpreter.

**Comment by RicochetYT** [comment]
Non-modified clients are also capable of doing this. Detecting this would also detect legitimate players.
